app/src/lib/menu-update.ts

```diff
@@ -164,12 +166,29 @@ function getMenuState(state: IAppState): Map<MenuIDs, IMenuItemState> {
     menuStateBuilder.disable('pull')
   }
 
+  const welcomeScopedIds: ReadonlyArray<MenuIDs> = [
```

Comment:
In agreement on the **Preferences**, but shouldn't **About** be accessible no matter what state the app is in? Since it's a [standard macOS menu item](https://electron.atom.io/docs/api/menu/#standard-menu-item-actions), I'd say yes.